internal/controller/numaresourcesoperator_controller.go

```diff
@@ -195,7 +195,12 @@ func updateStatusConditionsIfNeeded(instance *nropv1.NUMAResourcesOperator, cond
 		return
 	}
 	klog.InfoS("updateStatus", "condition", cond.Type, "reason", cond.Reason, "message", cond.Message)
-	conditions, ok := status.ComputeConditions(instance.Status.Conditions, cond.Type, cond.Reason, cond.Message)
+	conditions, ok := status.ComputeConditions(instance.Status.Conditions, metav1.Condition{
```

Comment:
please wait for my PR to merge, need to make sure the e2e tests work OK and cover the changes